R/expr-expr.R

```diff
@@ -364,12 +893,109 @@ expr__over <- function(
   })
 }
 
+#' Filter a single column.
+#'
+#' Mostly useful in an aggregation context. If you want to filter on a
+#' DataFrame level, use `DataFrame$filter()` (or `LazyFrame$filter()`).
+#'
+#' @param predicate An Expr or something coercible to an Expr. Must return a
+#' boolean.
+#'
+#' @inherit as_polars_expr return
+#'
+#' @examples
+#' df <- pl$DataFrame(
+#'   group_col = c("g1", "g1", "g2"),
+#'   b = c(1, 2, 3)
+#' )
+#' df
+#'
+#' df$group_by("group_col")$agg(
+#'   lt = pl$col("b")$filter(pl$col("b") < 2),
+#'   gte = pl$col("b")$filter(pl$col("b") >= 2)
+#' )
 expr__filter <- function(...) {
   parse_predicates_constraints_into_expression(...) |>
     self$`_rexpr`$filter() |>
     wrap()
 }
 
+## TODO Better explain aggregate list
+
+#' Map an expression with an R function
```

Comment:
Set noRd for now as map_batches is broken and will probabilistically stop the workflow when run on CI.